bokeh/command/subcommands/serve.py

```diff
@@ -59,6 +59,25 @@
 
 will have the Bokeh server listen all available network addresses.
 
+Additionally, it is possible to configure a hosts whitelist that must be
+matched by the ``Host`` header in new requests. You can specify multiple
+acceptable host values with the ``--host`` option:
+
+.. code-block:: sh
+
+    bokeh serve app_script.py --host foo.com:8081 --host bar.com
+
+If no port is specified in a host value, then port 80 will be used. In
+the example above Bokeh server will accept requests from ``foo.com:808``
+and ``bar.com:80``.
+
+If no host values are specified, then by default the Bokeh server will
+accept requests from ``localhost:<port>`` where ``<port>`` is the port
+that the server is configure to listen on.
```

Comment:
I had to think about this for a minute and it might be helpful for new comers to note that the default is 5006.